components/automate-gateway/gateway/services.go

```diff
@@ -351,7 +352,6 @@ func unversionedRESTMux(grpcURI string, dopts []grpc.DialOption) (http.Handler,
 		"gateway":              pb_gateway.RegisterGatewayHandlerFromEndpoint,
 		"legacy":               pb_legacy.RegisterLegacyDataCollectorHandlerFromEndpoint,
 		"license":              pb_license.RegisterLicenseHandlerFromEndpoint,
-		"authz":                pb_authz.RegisterAuthorizationHandlerFromEndpoint,
```

Comment:
Remove from the `/api/v0` top-level path component here; add to the `/apis` path component in L378 below.